src/main/java/com/facebook/openwifirrm/ucentral/UCentralUtils.java

```diff
@@ -440,4 +440,53 @@ public static String getBandFromChannel(int channel) {
 		}
 		return null;
 	}
+
+	/**
+	 * Tries to parse channel width, if it encounters an error it will return null.
+	 * It can handle 80p80 in two ways. First it can just treat it as 160. Second,
+	 * it can just apply to the first 80 channel and ignore the second. This is
+	 * controlled by treatSeparate. For purposes of TPC algorithms, we treat it as
+	 * 160 while channel assignment treats it as separate.
```

Comment:
This last sentence shouldn't be here, avoid putting comments in places where it'll get easily forgotten/outdated